File: libs/net/entities/ScheduleType.cs
namespace TNO.Entities;

/// <summary>
/// Provides schedule type.
/// </summary>
public enum ScheduleType
{
    /// <summary>
    /// This schedule does not run.
    /// </summary>
    None = 0,

    /// <summary>
    /// Continually runs.
    /// </summary>
    Continuous = 1,

    /// <summary>
    /// Daily runs a single start and stop event.
    /// </summary>
    Daily = 2,

    /// <summary>
    /// Runs based on a programmed schedule of multiple start ands top events.
    /// </summary>
    Advanced = 3,
}

File: app/editor/src/hooks/api-editor/constants/ScheduleType.ts
export enum ScheduleType {
  /** Repeating will run a continuously repeating schedule */
  Continuous = 1,
  /** Managed will run between a start and stop time */
  Daily = 2,
  /** Runs based on a programmed schedule of multiple start ands top events. */
  Advanced = 3,
}
